config/initializers/zz.rb

```diff
@@ -0,0 +1,11 @@
+if Rails.env.development?
+  # make sure we do not regress into slow startup time by preloading to much
+  Rails.configuration.after_initialize do
+    [
+      ActiveRecord::Base.send(:descendants).map(&:name),
+      ActionController::Base.descendants,
+      (File.basename($0) != "rake" && defined?(Rake) && "rake"),
```

Comment:
it's so that the raise says "rake should not be preloaded" :)

On Thu, Mar 19, 2015 at 12:04 PM, Shane Hender <notifications@github.com>
wrote:

> In config/initializers/zz.rb
> <https://github.com/zendesk/samson/pull/309#discussion_r26744541>:
>
> > @@ -0,0 +1,11 @@
> > +if Rails.env.development?
> > +  # make sure we do not regress into slow startup time by preloading to much
> > +  Rails.configuration.after_initialize do
> > +    [
> > +      ActiveRecord::Base.send(:descendants).map(&:name),
> > +      ActionController::Base.descendants,
> > +      (File.basename($0) != "rake" && defined?(Rake) && "rake"),
>
> Isn't the 3rd "rake" conditional obsolete since it'll always be 'true'?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/309/files#r26744541>.
>